app/user/rest.py

```diff
@@ -269,12 +269,11 @@ def verify_user_code(user_id):
 def send_user_2fa_code(user_id, code_type):
     user_to_send_to = get_user_by_id(user_id=user_id)
 
-    if verify_within_time(user_to_send_to, age=timedelta(seconds=10)) >= 1:
-        raise InvalidRequest("Code already sent, wait 10 seconds", status_code=400)
-
```

Comment:
Digging into github's commit history to know of the original intent of this, I saw that @maxneuvians actually added this. I am not sure if he will remember the reason but it could be good to sync with him to know why we thought this was necessary. The [change comes from here](https://github.com/cds-snc/notification-api/commit/b5c184a81522b3010730d2e72867745c700f4248).

Reply:
https://github.com/cds-snc/notification-api/pull/164 - basically people were jamming the login button and because there is a delay in SMS messages coming to phones they would get multiple ones and would never know which one is correct.

Reply:
Yes, so my change would prevent a new code from being sent, without raising an error